deploy/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
   {{- if .Values.ingress.dashboard.annotations }}
   annotations: {{- toYaml .Values.ingress.dashboard.annotations | nindent 4 }}
   {{- end }}
+    {{- if .Values.cephClusterSpec.dashboard.ssl }}
```

Comment:
The CI has this error:
```
Error: INSTALLATION FAILED: YAML parse error on rook-ceph-cluster/templates/ingress.yaml: error converting YAML to JSON: yaml: line 8: mapping values are not allowed in this context: exit status 1
```
This doesn't seem to work since on line 8 the annotations are already loaded from the dashboard.annotations.
Can you just add these annotations to your `ingress.dashboard.annotations`?

Reply:
It should be fixed now.